PolyPhen_SIFT.pm

```diff
@@ -28,8 +28,18 @@ limitations under the License.
 =head1 SYNOPSIS
 
  mv PolyPhen_SIFT.pm ~/.vep/Plugins
+
+ # Read default PolyPhen/SIFT SQLite file in $HOME/.vep
  ./vep -i variations.vcf -cache --plugin PolyPhen_SIFT
 
+ # Read database with custom name and/or located in a custom directory
+ ./vep -i variations.vcf -cache --plugin PolyPhen_SIFT,db=custom.db
+ ./vep -i variations.vcf -cache --plugin PolyPhen_SIFT,dir=/some/custom/dir
+ ./vep -i variations.vcf -cache --plugin PolyPhen_SIFT,db=custom.db,dir=/some/custom/dir
+
+ # Create PolyPhen/SIFT SQLite file based on Ensembl database
+ ./vep -i variations.vcf -cache --plugin PolyPhen_SIFT,create_db=1
```

Comment:
Hi @nuno-agostinho , the PR looks good. Just a small question, can we still use cache when we want to create_db=1 since we enable database?

Reply:
Hi! VEP (including the PolyPhen/SIFT plugin) runs as normal when using `create_db=1`; after all, `--cache` still allows for database connection if needed.

The only issue would come with using `--offline` because the database connection will error out, but the documentation already warned about that in previous versions of the plugin.